pylint/config.py

```diff
@@ -593,8 +616,12 @@ def parse(self, to_parse, config):
                 config.set_option(option, value)
 
     def write(self, stream=sys.stdout):
-        # TODO: Check if option descriptions are written out
-        self._parser.write(stream)
+        with six.StringIO() as temp_stream:
+            self._parser.write(temp_stream)
```

Comment:
I noticed some differences in the output between `configparser` and the old `generate_config`.

- There are spaces around the `=`.
- Booleans are `y` instead of `yes`.
- Comma separated lists are all on one line instead of vertically aligned.
- The order of options is different.
- There is less newline whitespace around sections options and comments.
- Missing options are omitted instead of commented out.
- There's a section called DEFAULT instead of MASTER.

I did the minimal implementation. Does any of that need adding back?

Reply:
Here's a diff between `pylint --generate-rcfile` from latest release and this branch https://gist.github.com/fffergal/43d79ebad767db41a0bbe40c909ec7c0.

Reply:
Yes, we definitely want back at least the vertically aligned comma separated lists and the `MASTER` section.

Reply:
OK, I'll look into a revision that revives `generate_config`.